lib/Translator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Translator
+{
+    private const DEFAULT_LANGUAGE = 'english';
+
+    public static function loadTranslations(): void
+    {
+        global $CONFIG, $_LANG;
+
+        if (isset($_SESSION['Language'])) {
+            $language = $_SESSION['Language'];
+        } elseif (isset($_SESSION['adminlang'])) {
+            $language = $_SESSION['adminlang'];
+        } else {
+            $language = $CONFIG['Language'];
+        }
+
+        if (!in_array($language, self::getAvailableLanguages())) {
+            $language = self::DEFAULT_LANGUAGE;
+        }
+
+        require self::getLanguageDir() . '/' . $language . '.php';
+    }
+
+    public static function translate(string $key): string
+    {
+        global $_LANG;
+
+        return isset($_LANG[$key]) ? $_LANG[$key] : $key;
```

Comment:
Depends on the PHP version you could use
```suggestion
        return $_LANG[$key] ?? $key;
```

Reply:
Done